### PR TITLE
Automate testing Plugins view from the IDE

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Automate testing Plugins view from the IDE

### Screenshot/screencast of this PR

[test.log](https://github.com/eclipse/che/files/6440406/test.log)

### What issues does this PR fix or reference?

Issue: https://github.com/eclipse/che/issues/19639

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/e2e/pageobjects/ide/TopMenu.ts [hunk 1]
```diff
@@ -46,7 +44,6 @@ export class TopMenu {
 
         const buttonLocator: By = this.getTopMenuButtonLocator(buttonText);
 
-        await this.ide.closeAllNotifications();
```
Comment (ScrewTSW) [inline tests/e2e/pageobjects/ide/TopMenu.ts:49]: @Ohrimenko1988 any reason why this one was removed?

Comment (Ohrimenko1988) [reply tests/e2e/pageobjects/ide/TopMenu.ts:49]: As far as I see this is a redundant step which were implemented in the times when notification was displaying at the top of the screen. And notification could overlap the top menu. Currently it is impossible so no reasons to keep this step. Besides it leads to errors from time to time.

**Comment by Ohrimenko1988** [comment]
[crw-ci-test --rebuild]

File: tests/e2e/pageobjects/ide/TopMenu.ts (hunk 1 above)
Comment (ScrewTSW) [reply tests/e2e/pageobjects/ide/TopMenu.ts:49]: sounds reasonable 👍

**Review by ScrewTSW — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
[crw-ci-test --rebuild]

**Comment by Ohrimenko1988** [comment]
[crw-ci-test --rebuild]

**Pull request merged**